src/resources/editor/tools/yaml/yaml-intelligence-resources.json

```diff
@@ -12496,6 +12498,8 @@
     "The input document that will serve as the root document for this\nmanuscript",
     "The deployed url for this manuscript",
     "Whether to generate a MECA bundle for this manuscript",
+    "Additional file resources to be copied to output directory",
```

Comment:
@dragonstyle I'm not sure why I picked this up.  Should I remove it?

Reply:
I'm not sure either- I definitely added that but I _thought_ I regenerated the underlying schema and committed that alongside. I must not have! Safe to ignore.

Reply:
This may be my workflow now.  If I see conflicts, I just accept "theirs" in my IDE rather than a manual resolve.  After that I run "build-js".  I'm hoping that my changes are just replaced, then I regenerate my changes back in.  I'm going to try this again and see if that resolves it.  Thanks for reviewing.

Reply:
This seems to have resolved itself.